### DISCUSSION TITLE
SCREEGO_TURN_PORT_RANGE

### DISCUSSION THREAD

**Discussion by christoph-wing** [screego/server#203] [Q&A] [answered]
Hello, a question about ``SCREEGO_TURN_PORT_RANGE``:
Do I understand correctly that one port is required per user at the same time (of course only if TURN is used instead of STUN)? My hoster only lets me open ports individually, so that would be good to know. So if I open ``50000-50010``, for example, would that be enough for 10 parallel users or how should I understand this?
Thanks for your help.

**Comment by jmattheis** [comment, marked as answer]
It's at least `shared video streams * 2 * users in the room` connections. It seems currently it's also another `* 2` which may be a bug in screego. Additionally there is a bug in the turn server where not all connection are properly cleaned in some cases https://github.com/pion/turn/issues/246. So you may need some more.
